wakuv2/api.go

```diff
@@ -376,27 +363,15 @@ func (api *PublicWakuAPI) Messages(ctx context.Context, crit Criteria) (*rpc.Sub
 	return rpcSub, nil
 }
 
-// Message is the RPC representation of a waku message.
-type Message struct {
-	Sig          []byte           `json:"sig,omitempty"`
-	Timestamp    uint32           `json:"timestamp"`
-	PubsubTopic  string           `json:"pubsubTopic"`
-	ContentTopic common.TopicType `json:"topic"`
-	Payload      []byte           `json:"payload"`
-	Padding      []byte           `json:"padding"`
-	Hash         []byte           `json:"hash"`
-	Dst          []byte           `json:"recipientPublicKey,omitempty"`
-}
-
 // ToWakuMessage converts an internal message into an API version.
-func ToWakuMessage(message *common.ReceivedMessage) *Message {
-	msg := Message{
-		Payload:      message.Data,
-		Padding:      message.Padding,
-		Timestamp:    message.Sent,
-		Hash:         message.Hash().Bytes(),
-		PubsubTopic:  message.PubsubTopic,
-		ContentTopic: message.ContentTopic,
+func ToWakuMessage(message *common.ReceivedMessage) *wakuinterface.Message {
+	msg := wakuinterface.Message{
+		Payload:     message.Data,
+		Padding:     message.Padding,
+		Timestamp:   message.Sent,
+		Hash:        message.Hash().Bytes(),
+		PubsubTopic: message.PubsubTopic,
+		Topic:       types.TopicType(message.ContentTopic),
```

Comment:
Why `Topic` instead of `ContentTopic`?

Reply:
nvm, I just saw that we're reusing the `ReceivedMessage`, and renamed it to `Message`. Probably a good moment to rename that property to `ContentTopic`, as it is easier to know what kind of topic are we dealing with!